src/Nut/DebugEvents.php

```diff
@@ -21,6 +23,7 @@ protected function configure()
         $this
             ->setName('debug:events')
             ->setDescription('System events, and target callable debug dumper.')
+            ->addOption('sort-listener', null, InputOption::VALUE_NONE, 'Sort events in order of listener name.')
```

Comment:
I guess so. In that case `sort-name` makes more sense. Same for service providers command.